tensorflow/core/profiler/internal/gpu/device_tracer_test.cc

```diff
@@ -266,8 +266,13 @@ TEST_F(DeviceTracerTest, TraceToXSpace) {
   XSpace space;
   TF_ASSERT_OK(tracer->CollectData(&space));
   // At least one gpu plane and one host plane for launching events.
+#if GOOGLE_CUDA
```

Comment:
Instead of adding macros within the test (inline macros hurts readability), please duplicate this and have a specialized test for ROCM thath is guarded by TENSORFLOW_USE_ROCM.

Reply:
done